source/index.html.md

```diff
@@ -0,0 +1,429 @@
+
+The Application contains various services: Project Service, Employee Service and Logged Hours Service which use same Database.
+
+The Project Service has various functionalities such as add project, list all project, get project by employee ID and delete project
+
+
+
+
+# User 
+```shell
+#FORMAT OF DATA 
+```
+Represents user details.
+
+## User attributes:
+>
+Employee data format:
+>
+
+```json
+
+  {
+  "employee_id": 0,
+  "employee_firstname": "string",
+  "employee_lastname": "string",
+  "employee_email": "string",  
+ } 
+```
+
+
+Name | Type | Description
+---------|---------- | -----------
+employee_id | Number | unique identifier for employee. 
+employee_firstname | String | First Name. 
+employee_lastname | String | Last Name.
+employee_email | String | Email.
+
+
+> 
+Project data format:
+>
+```json
+ {
+  "project_id": 0,
+  "project_date": "date",
+  "project_hours": "string",
+  "sick_date": "date",
+  "vacation_date": "date",
+ }
+```
+
+
+
+Name |Type| Description
+--------- | ----------- | -----------
+project_id | Number | unique identifier for project
+project_date | Date | Date of project
+project_hours | Number | Hours.
+
+
+Name |Type | Description
+--------- | ----------- | -----------
+sick_date | Date | Date for sick log.
+vacation_date | Date | Date for vacation log.
+
+
+
+# Authentication
+
+> To authorize, use this code:
+
+
+```shell
+curl "https://project.services.turntabl.io/" 
+```
+
+> Make sure you are a Turntabl employee before assess this API.
+
+TPMS uses SAML to allow access to the API. You can only be registered a new Turntabl employee by the Administrator, TPMS API at our [developer portal](http://turntabl.io/developers).
+
+
+
+<aside class="notice">
+Make sure you are a Turntabl employee before assess this API
+</aside>
+
+# Project  Controller
+
+## Get All Projects
+
+
+```shell
+curl "https://project.services.turntabl.io/v1/api/projects" 
+-H  "accept: */*"  
+```
+
+
+> The above command returns JSON structured like this:
+
+```json
+
+  {
+    "project_id": 1,
+    "project_name": "TPMS WEB APP"    
+  },
+  {
+   "project_id": 2,
+    "project_name": "AWS"   
+  }
+
+```
+
+This endpoint is used to retrieve all projects.
+
+### HTTP Request
+
+
+`GET https://project.services.turntabl.io/v1/api/projects`
+
+
+
+## Add New Project
+
+
+```shell
+curl "https://project.services.turntabl.io/v1/api/project" 
+-H  "accept: */*" 
+-H  "Content-Type: application/json" 
+-d "{  \"project_name\": \"string\"}"
+```
+
+
+> The above command returns JSON structured like this:
+
+```json
+
+  {   
+    "project_name": "string"    
+  }
+
+```
+
+This endpoint is used to add new project.
+
+### HTTP Request
+
+
+`POST https://project.services.turntabl.io/v1/api/project`
+
+
+
+
+
+
+
+## Get Projects Assigned to Employee By His/Her ID
+
+
+```shell
+curl "https://project.services.turntabl.io/v1/api/projects/assigned/employee/id" 
+-H  "accept: */*"
```

Comment:
can you kill these newlines so I can c&p into a console?

Reply:
done